src/main/java/com/epam/preprod/sirenko/command/CommandContainer.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.preprod.sirenko.command;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * All commands holder
+ */
+public class CommandContainer {
```

Comment:
Please create directory for all containers and move other containers to is as well as CommandContainer

Reply:
done